Dockerfile

```diff
@@ -24,7 +24,7 @@ RUN bash install-minisign.sh
 
 # Install zig
 # https://ziglang.org/download/
-ARG ZIG_VERSION="0.13.0"
+ARG ZIG_VERSION="0.14.0"
```

Comment:
https://ghostty.org/docs/install/build

Docs says we should use Zig 0.14.1

Reply:
Nice, thanks!